Tests/MeshLib/TestAddLayerToMesh.cpp

```diff
@@ -31,9 +31,9 @@ namespace AddLayerValidation
         ElementErrorFlag const flags[nErrorFlags] = {ElementErrorFlag::ZeroVolume,
         ElementErrorFlag::NonCoplanar, ElementErrorFlag::NonConvex,  ElementErrorFlag::NodeOrder};
         std::vector<ElementErrorCode> const codes (MeshLib::MeshValidation::testElementGeometry(mesh));
-        for (std::size_t i=0; i<codes.size(); ++i)
+        for (auto code : codes)
```

Comment:
It's a the `code` is a `std::bitset<5>`, so no reference like for doubles or ints.